pipe_segment/transform/fragment.py

```diff
@@ -12,6 +12,9 @@
 logger.setLevel(logging.DEBUG)
 
 
+def key_or_none(d, k):
+    return d[k] if k in d else None
+
```

Comment:
There's a [dict get method](https://docs.python.org/3/library/stdtypes.html#dict.get) that does exactly this.